rasa/shared/nlu/training_data/entities_parser.py

```diff
@@ -141,12 +142,10 @@ def get_validated_dict(json_str: Text) -> Dict[Text, Text]:
     try:
         data = json.loads(f"{{{json_str}}}")
     except JSONDecodeError as e:
-        rasa.shared.utils.io.raise_warning(
+        raise InvalidEntityFormatException(
             f"Incorrect training data format ('{{{json_str}}}'). Make sure your "
-            f"data is valid.",
-            docs=DOCS_URL_TRAINING_DATA_NLU,
-        )
-        raise e
+            f"data is valid. More info at {DOCS_URL_TRAINING_DATA_NLU}"
+        ) from e
 
     validation_utils.validate_training_data(data, schema.entity_dict_schema())
```

Comment:
what about the `ValidationError` here? Should in my opinion also raise a `InvalidEntityFormatException`

Reply:
I'm afraid it can break some behaviours, especially the `RasaYAMLReader`'s. I'd rather rename the exception to something more explicit if it's not already the case: the parsing broke because of an invalid format (not JSON). Not because the format didn't follow a specific schema (ValidationError). WDYT?

Reply:
yes better 👍🏻 I've made the changes in ffb283f1d45